httpClient/client.go

```diff
@@ -0,0 +1,86 @@
+// Package Create an HTTP client in Go that consumes the datetime server APIs implemented in [Datetime Saerver](codescalersinternships/home#284)
+
+package httpClient
+
+import (
+	"fmt"
+	"github.com/joho/godotenv"
+	"github.com/sirupsen/logrus"
+	"net/http"
+	"os"
+	"time"
+)
+
+// Client represent a http client
+type Client struct {
+	Client *http.Client
+	Url    string
+}
+
+// Generic http client for default values
+var (
```

Comment:
default values should be const, having them as variables and exported can lead to the user updating them in an unexpected way